visual-python/src/saucelabs_visual/frameworks/robot.py

```diff
@@ -265,7 +304,7 @@ def visual_snapshot(
 
         self.client.create_snapshot_from_webdriver(
             name=name,
-            session_id=session_id,
+            driver=self._get_driver(),
```

Comment:
This is technically a breaking change if anyone is using the current framework-free Python implementation. We're requiring the job ID here anyway -- so it would require manual intervention regardless. I figured it would be better to do this now and bypass all of that and just do a minor bump since this is prerelease.